src/main/java/com/cleanroommc/groovyscript/compat/mods/prodigytech/PrimordialisReactor.java

```diff
@@ -48,6 +51,14 @@ private boolean removeRecipeBase(IIngredient x) {
 
     @MethodDescription(example = @Example("item('minecraft:diamond')"), type = MethodDescription.Type.ADDITION)
     public void add(IIngredient x) {
+        if (IngredientHelper.overMaxSize(x, 1)) {
+            // PT modifies the recipe to only consume 1 item
+            GroovyLog.Msg msg = GroovyLog.msg("Error adding Primordialis Reactor fuel").error();
+            msg.add("Expected input stack size of 1");
+            msg.post();

```

Comment:
should chain in these situations instead 
```suggestion
            GroovyLog.msg("Error adding Primordialis Reactor fuel").error()
                    .add("Expected input stack size of 1")
                    .post();
```